gsa/src/web/pages/tasks/listpage.js

```diff
@@ -342,6 +354,16 @@ const TasksListPage = () => {
             onCloseClick={closeNotificationDialog}
           />
           <Download ref={downloadRef} />
+          {tagsDialogVisible && (
+            <BulkTagComponent
```

Comment:
Currently I am not sure if this will really open the dialog because when mounting the BulkTagComponent the dialog visibility state is set to false.

Reply:
Oh I got confused by the different dialogs. We have a second sub dialog for creating a new tag. The Tags dialog is opened correctly. So this is fine!